client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -185,36 +165,33 @@ function HelpSearchResults( {
 					{ results.map( renderHelpLink ) }
 				</ul>
 			</Fragment>
-		);
+		) : null;
 		/* eslint-enable jsx-a11y/no-noninteractive-element-to-interactive-role */
 	};
 
-	const renderSearchSections = ( results, query ) => {
-		// Get the unique result types
-		// TODO: Clean this up. There has to be a simpler way to find the unique search result types
-		const searchResultTypes = results
-			.map( ( searchResult ) => searchResult.support_type )
-			.filter( ( type, index, arr ) => arr.indexOf( type ) === index );
-
-		return searchResultTypes.map( ( resultType ) => {
-			let displayedResults = results;
-			switch ( resultType ) {
-				case SUPPORT_TYPE_CONTEXTUAL_HELP:
-					displayedResults = results.filter( ( r ) => r.support_type === resultType ).slice( 0, 6 );
-					break;
-				case SUPPORT_TYPE_API_HELP:
-					displayedResults = results.filter( ( r ) => r.support_type === resultType ).slice( 0, 5 );
-					break;
-				case SUPPORT_TYPE_ADMIN_SECTION:
-					displayedResults = results.filter( ( r ) => r.support_type === resultType ).slice( 0, 3 );
-					break;
-			}
-			return renderSearchResultsSection(
-				`inline-search--${ resultType }`,
-				getTitleBySectionType( resultType, query ),
-				displayedResults
-			);
-		} );
+	const renderSearchSections = () => {
+		const sections = [
+			{
+				id: `inline-search--${ SUPPORT_TYPE_API_HELP }`,
+				title: translate( 'Support articles' ),
+				results: searchResults.slice( 0, 5 ),
+				condition: ! isSearching && searchResults.length > 0,
+			},
+			{
+				id: `inline-search--${ SUPPORT_TYPE_CONTEXTUAL_HELP }`,
+				title: ! searchQuery.length ? translate( 'This might interest you' ) : '',
```

Comment:
please note, this was vice versa before but it makes much more sense to me this way